sdk/communication/communication-call-automation/swagger/README.md

```diff
@@ -0,0 +1,31 @@
+# Azure Communication Services Call Automation
+
+> see [https://aka.ms/autorest](https://aka.ms/autorest)
+
+## Configuration
+
+```yaml
+package-name: azure-communication-call-automation
+title: CallAutomationApiClient
+description: Call Automation Client
+generate-metadata: false
+license-header: MICROSOFT_MIT_NO_VERSION
+output-folder: ../src/generated
+tag: V2023-01-15-preview
+input-file: https://raw.githubusercontent.com/williamzhao87/azure-rest-api-specs/0d0cd5af40aa17af76ce0307ac5512351c38e3bc/specification/communication/data-plane/CallAutomation/preview/2023-01-15-preview/communicationservicescallautomation.json
+package-version: 1.0.0-beta.1
+model-date-time-as-string: false
+optional-response-headers: true
+typescript: true
+azure-arm: false
+add-credentials: false
+use-extension:
+  "@autorest/typescript": "latest"
+directive:
+- rename-model:
```

Comment:
we should try to rename all internal models that has public facing model with same name

Reply:
Min please help on doing this as my system cannot do the js autorest for unknown reason